scipy/stats/stats.py

```diff
@@ -3004,16 +3004,16 @@ def pearsonr(x, y):
 
     .. math::
 
-        r_{pb} = \frac{\sum (x - m_x) (y - m_y)}
-                      {\sqrt{\sum (x - m_x)^2 \sum (y - m_y)^2}}
+        r_{pb} = \frac{\sum_{i=1}^N (x_i - m_x) (y_i - m_y)}
+                {\sqrt{\sum_{i=1}^N (x_i - m_x)^2 \sum_{i=1}^N(y_i - m_y)^2}}
 
     where :math:`m_x` is the mean of the vector :math:`x` and :math:`m_y` is
     the mean of the vector :math:`y`.
 
 
     References
     ----------
-    http://www.statsoft.com/textbook/glosp.html#Pearson%20Correlation
+    .. [1] https://en.wikipedia.org/wiki/Pearson_correlation_coefficient#Testing_using_Student's_t-distribution
```

Comment:
Can we do something about the length of this link without using a URL shortener? I am not sure if linebreaks are allowed in the inline refs.